stylesheets/index.css

```diff
@@ -0,0 +1,106 @@
+
+/* 
+body.font-KetchumItalic {
+    font-family: 'KetchumItalic', sans-serif;
+}
+
+
+body.font-PokemonSolid {
+    font-family: 'PokemonSolid', sans-serif;
+}
+
+body.font-KetchumWeeV {
+    font-family: 'KetchumWeeV', sans-serif;
+}
+
+body.font-PokemonGb {
+    font-family: 'PokemonGb', sans-serif;
+}*/
+
+
+
+
+
+
+
+
+
+
+
+
+/* main {
+    padding: 20px;
+    margin: 10pk;
+    margin-left: 30px;
+    margin-right: 30px;
+    flex: 1;
+} */
+
+
+
+/* h2 {
+    color: #af95ff;
+    font-size: 24px;
+    margin-bottom: 10px;
+} */
+
+
+strong {
+    font-weight: bold;
+} 
+
+.section-container {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.section-boxed {
+    flex: 0 0 calc(50% - 30px);
+    border: 4px ridge #af95ff;
+    border-radius: 25px;
+    padding: 20px;
+    margin-bottom: 30px;
+    box-sizing: border-box; 
+}
+
+.section-boxed h2 {
+    color: #af95ff; 
+    margin-bottom: 10px;
+}
+
+.section-boxed ul {
+    list-style: disc;
+    margin-left: 20px;
+}
+
+section#about-adiemon {
```

Comment:
Unless you have an element directly following about-adiemon or you have multiple elements with the id "about-adiemon" (which you shouldn't), you don't need to include the section element here.